### PR TITLE
EBODAC-810 Ui Test

### PR DESCRIPTION
**Pull request by aczyrko**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by MikolajBasSoldevelo** [comment]
test this please

**Comment by MikolajBasSoldevelo** [comment]
test this please

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/test/ParticipantsWhoOptOutofReceivingMOTECHMessagesUiTest.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+import org.motechproject.ebodac.uitest.page.HomePage;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.motechproject.ebodac.uitest.page.*;
+
+import java.lang.Exception;
+
+/**
+ * Created by serwis on 09.02.16.
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/test/ParticipantsWhoOptOutofReceivingMOTECHMessagesUiTest.java:19]: remove this comment

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/test/ParticipantsWhoOptOutofReceivingMOTECHMessagesUiTest.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        L1adminUser = properties.getUserName();
+        L1adminpassword = properties.getPassword();
+        loginPage = new LoginPage(driver);
+        homePage = new HomePage(driver);
+        ebodacPage = new EBODACPage(driver);
+        url = properties.getWebAppUrl();
+
+            }
+        @Test //Test for EBODAC-810
+
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/test/ParticipantsWhoOptOutofReceivingMOTECHMessagesUiTest.java:47]: remove this empty line

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/test/ParticipantsWhoOptOutofReceivingMOTECHMessagesUiTest.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        public void participantsWhoOptOutofReceivingMOTECHMessagesUiTest ()throws InterruptedException {
+            httpClientHelper = new UITestHttpClientHelper(url);
+            if (homePage.expectedUrlPath() != currentPage().urlPath()) {
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/test/ParticipantsWhoOptOutofReceivingMOTECHMessagesUiTest.java:50]: improve formatting, you have to many white spaces

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/test/ParticipantsWhoOptOutofReceivingMOTECHMessagesUiTest.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+                loginPage.login(L1adminUser, L1adminpassword);
+                homePage.openEBODACModule();
+                ebodacPage.gotoReports();
+                reportPage.showParticipantWhoOptOutofReceivingMOTECHMessagesReport();
+                Thread.sleep(500);
+            }
+
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/test/ParticipantsWhoOptOutofReceivingMOTECHMessagesUiTest.java:57]: remove those unnecessary empty lines

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/test/BookingApplicationCapacityInfoUiTest.java [hunk 5]
```diff
@@ -52,7 +52,7 @@ public void bookingApplicationCapacityInfoTest() throws InterruptedException {
         bookingAppCapacityInfoPage.filterNext7Days();
         Assert.assertEquals(bookingAppCapacityInfoPage.getMaxCapacity(), ""+(7*maxCapacity));
         bookingAppCapacityInfoPage.filterDateRange();
-        Assert.assertEquals(bookingAppCapacityInfoPage.getMaxCapacity(), ""+(28*maxCapacity));
+        Assert.assertEquals(bookingAppCapacityInfoPage.getMaxCapacity(), ""+(29*maxCapacity));
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/test/BookingApplicationCapacityInfoUiTest.java:55]: add white space before and after + and *

**Pull request closed without merging**